src/mbgl/map/map_options.cpp

```diff
@@ -9,6 +9,7 @@ class MapOptions::Impl {
     ViewportMode viewportMode = ViewportMode::Default;
     NorthOrientation orientation = NorthOrientation::Upwards;
     bool crossSourceCollisions = true;
+    Size size;
```

Comment:
:thinking: A defaulted constructed `Size` is empty, so constructing a map with a default constructed `MapOptions` would cause `Transform::resize` to throw. Should we have a different default value?